styles/zeiss-styles.css

```diff
@@ -1872,3 +1872,32 @@ sub {
   background-color: var(--border-color-active);
   height: 0.125rem
 }
+
```

Comment:
These styles should go in `styles.css` as `zeiss-styles.css` is a temporary file till we get done with markup simplification.